site-manage/pageorder/tool/src/java/org/sakaiproject/site/tool/helper/order/impl/SitePageEditHandler.java

```diff
@@ -151,6 +156,11 @@ public void init() {
                 unhideables.add(toolIds[i].trim());
             }
         }
+        String uneditablesConfig = serverConfigurationService.getString(UNEDITABLES_CFG, "");
```

Comment:
Yep, I was just following the pattern of `poh.unhideables` which puts them all on one line.

Personally I'm not very keen on the `ServerConfigrationService.getString(String)` for simple entries as for something like this you end up having:
```
poh.uneditables.length=2
poh.uneditables.1=sakai.siteinfo
poh.uneabledles.2=sakai.polls
```
which is easy to get wrong when writing compared to:
```
poh.uneditables=sakai.siteinfo,sakai.polls
```
However for complex entries that might have commas in them or span multiple lines I think it's the getStrings is really good.